source/digits_hits/include/GateGlobalActor.hh

```diff
@@ -309,6 +333,26 @@ private:
 	 * */
 	void UpdateProcessName(const G4Step& step);
 
+	/** Update data about parent ID
+	 * */
+	void UpdateParentID(const G4Step& step);
+
+	/** Update interaction time value
+	 * */
+	void UpdateInteractionTime(const G4Step& step);
+
+	/** Update local time
```

Comment:
I really think those comment are redunand, since you chose good names for your functions

Reply:
http://www.informit.com/articles/article.aspx?p=1327761

Reply:
I agree. I will remove methods descriptions, but leave variables descriptions - in my opinion variables descriptions are critical for code understanding.